mypy_django_plugin/transformers/managers.py

```diff
@@ -71,25 +71,38 @@ def get_method_type_from_dynamic_manager(
     queryset_info = helpers.lookup_fully_qualified_typeinfo(api, queryset_fullname)
     assert queryset_info is not None
 
-    def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[ProperType]:
-        # TODO: Handle @overload?
-        if isinstance(definition, FuncBase) and not isinstance(definition, OverloadedFuncDef):
-            return definition.type
-        elif isinstance(definition, Decorator):
-            return definition.func.type
-        return None
+    is_fallback_queryset = queryset_info.metadata.get("django", {}).get("any_fallback_queryset", False)
 
-    method_type = get_funcdef_type(queryset_info.get_method(method_name))
+    method_type = _get_funcdef_type(queryset_info.get_method(method_name))
     if method_type is None:
         return None
 
-    assert isinstance(method_type, CallableType)
+    assert isinstance(method_type, FunctionLike)
```

Comment:
Can `_get_funcdef_type` return `FunctionLike`? To get rid of the assert, or can we branch on it instead of asserting?

Reply:
Changed to `if`. Because it can return `Any` for sure.